### PR TITLE
Adds the ability to select which language faker will generate the data

### PR DESCRIPTION
**Pull request by rleger**
It would be nice to be able to select which language Faker will generate the factory data from.

I thought of using the app.locale var but what if the language is not available in faker... might be cleaner to differentiate the two.

So instead I thought you could add a dedicated config option in the database file for instance. That way you can swap it easily.

Sorry if this is not very clean, just trying to contribute in my own little way with a solution to something that's bothering me.

Thank you.

config/database.php

```
    /*
    |--------------------------------------------------------------------------
    | Faker locale
    |--------------------------------------------------------------------------
    |
    | Language in which faker will generate dummy data
    |
    */

    'faker_locale' => 'fr_FR', 
```

### CODE REVIEW

File: src/Illuminate/Database/DatabaseServiceProvider.php [hunk 1]
```diff
@@ -60,7 +60,7 @@ public function register()
     protected function registerEloquentFactory()
     {
         $this->app->singleton(FakerGenerator::class, function () {
-            return FakerFactory::create();
+        	return FakerFactory::create(config('database.faker_locale', 'en_US'));
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/DatabaseServiceProvider.php]: please don't change the indentation

Comment (GrahamCampbell) [inline src/Illuminate/Database/DatabaseServiceProvider.php]: you can't use the config helper function from a service provider not dependent on the foundation helpers

**Comment by rleger** [comment]
Of course.. sorry, removed the helper function call and replaced it with $app['config']. Sorry about the indentation, I got an error from StyleCI. Thanks

**Comment by taylorotwell** [comment]
I really prefer to not have another config option for this.

**Comment by rleger** [comment]
ok, how about a "service" that would try to initialize faker with the language set in app.locale and default to en_US ? Right now I'm overriding the binding in my own AppServiceProvider which is fine but a bit hacky.

**Comment by taylorotwell** [comment]
I don’t think that’s really hacky at all. That’s the purpose of the IoC container.

**Comment by rleger** [comment]
alright, fine by me. Thanks for your reply

**Pull request closed without merging**